packages/commands/src/index.ts

```diff
@@ -1167,6 +1197,7 @@ namespace Private {
       dataset: asFunc(options.dataset, emptyDatasetFunc),
       isEnabled: options.isEnabled || trueFunc,
       isToggled: options.isToggled || falseFunc,
+      isToggleable: options.isToggleable || !!options.isToggled,
```

Comment:
Is this correct? It seems like it should only be `!!options.isToggled` and never use the `options.isToggleable`.